js/bump_version.sh

```diff
@@ -5,5 +5,5 @@
 
 set -e
 SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
-NPM_VERSIONS=`npm show rive-js versions`
```

Comment:
This is the main fix, we need versions for the different repos, not just rive-js